# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/trackballs/hotfix-sdlttf.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- trackballs-1.3.4/src/glHelp.h.vanilla	2025-07-13 21:59:29.560492873 +0200
+++ trackballs-1.3.4/src/glHelp.h	2025-07-13 22:00:07.460965248 +0200
@@ -27,7 +27,7 @@
 
 #include "general.h"
 
-typedef struct _TTF_Font TTF_Font;
+typedef struct TTF_Font TTF_Font;
 typedef struct SDL_Surface SDL_Surface;
 class Map;
 class Game;
